frontend/public/components/factory/table.tsx

```diff
@@ -115,8 +116,8 @@ const sorts = {
   targetScrapeDuration: (target: Target): number => target?.lastScrapeDuration,
   string: (val) => JSON.stringify(val),
   number: (val) => _.toNumber(val),
-  getClusterOperatorStatus,
-  getClusterOperatorVersion,
+  getClusterOperatorStatus: (operator: ClusterOperator) => getClusterOperatorStatus(operator),
+  getClusterOperatorVersion: (operator: ClusterOperator) => getClusterOperatorVersion(operator),
```

Comment:
Is there an issue for other sorts like `podPhase` above?

Reply:
No as the `Pods` table was updated to the new virtualized table component with https://github.com/rhamilto/console/commit/33f02a8f454519e10fd498a3037e25c26f3d0c0a.  Will check the other `sorts` using the shorthand and file separate bugs if I find any.

Reply:
I did not find any other active [1] `sortFunc` sorts that do not appear to be working correctly.

[1] I did not test `serviceCatalogStatus`, `serviceClassDisplayName`, etc. since the service catalog is no longer supported.

Reply:
I'm debating if we want to proactively update the other similar sorts in this file to avoid the problem in the future. We should be able to remove the pod table sorts if we're no longer used as well. Do you mind opening a follow-on bug or issue?

Reply:
Or if it's easier we can update this PR.

Reply:
I prefer to keep this PR focused on the bug as it is filed and open a new tech debt issue to address the others.

Reply:
Works for me.

Reply:
New issue created:  https://issues.redhat.com/browse/HAC-711